slides/src/content/component.md

```diff
@@ -61,14 +61,14 @@ export class ToDoListComponent implements OnInit {
 
 ## Including a Component
 
-- Refer to component by putting its selector tag in something else's HTML
-  - Remove the completed list and summary to make things fit on the slide
+- Refer to the component by putting its `selector` in another component's template
 
 _src/app/app.component.html_
 ```html
 <h1>{{title}}</h1>
+<p>Summary: {{summary()}}</p>
 <p *ngFor="let item of thingsToDo; let i = index" id="{{i}}">({{i}}) {{item}}</p>
-<app-to-do-list></app-to-do-list>
+<p *ngIf="thingsCompleted.length == 0">Nothing completed</p>
```

Comment:
Has this one been brought in by accident? Doesn't seem related to the previous change.